File: CsharpProjects/AbstractFactorySolucao/AbstractFactorySolucao/Boleto/Familia/CalculosFactory.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using AbstractFactorySolucao.Boleto.ConfiguraJuros;
using AbstractFactorySolucao.Boleto.ConfiguraDesconto;
using AbstractFactorySolucao.Boleto.ConfiguraMulta;

namespace AbstractFactorySolucao.Boleto.Familia
{
    public interface CalculosFactory
    {
        public Juros criarJuros();
        public Desconto criarDesconto();
        public Multa criarMulta();

    }
}

File: JavaProjects/AbstractFactorySolucao/src/boleto/familia/CalculosFactory.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Interface.java to edit this template
 */
package boleto.familia;

import boleto.configuraDesconto.Desconto;
import boleto.configuraJuros.Juros;
import boleto.configuraMulta.Multa;

/**
 *
 * @author jeffe
 */
public interface CalculosFactory {
    
    public Juros criarJuros();
    public Desconto criarDesconto();
    public Multa criarMulta();
    
}
